htdocs/web_portal/views/user/view_user.php

```diff
@@ -127,6 +129,21 @@
                         </td>
                     </tr>
                 <?php } ?>
+                <?php if ($isCallingUserAdmin) : ?>
+                <tr class="site_table_row_1">
+                    <td class="site_table">Last Logged In</td>
+                    <td class="site_table">
+                        <?php
+                        if ($params['user']->getLastLoginDate()) {
+                            xecho(
```

Comment:
what is xecho?

Reply:
It's a function to mark and say it is safe content @rowan04 . Please have a look at this file htdocs/web_portal/index.php. It has nice link attached to the function.

`function xecho($data) {
   echo xssafe($data);
}`